src/stackedBars.ts

```diff
@@ -0,0 +1,58 @@
+import * as d3 from "d3";
+import {Pareto} from "./pareto";
+
+/**
+ * Render the bars using d3
+ * @param pareto Pareto data structure
+ */
+ export function renderStackedBars(pareto: Pareto) {
+
+    const stackedBars = pareto.stackedBars;
+    
+    const margin = {"top": 20, "left": 20, "right": 20, "bottom": 20};
+
+    // select the svg
```

Comment:
Just a small suggestion on the style, I noticed on the D3 branch we have another way of selecting, using containers. For style unification purposes I would suggest to use it here as well 
// select the svg
    let svg = d3.select("svg");
    let svgContainer = document.querySelector("#svg");
    

    //Get width and height from svg. cast to Number 
    const width = svgContainer?.getBoundingClientRect().width as number;
    const height = svgContainer?.getBoundingClientRect().height as number;